File: 프로그래머스/lv0/120813. 짝수는 싫어요/짝수는 싫어요.java
class Solution {
    public int[] solution(int n) {
        int cnt = 0;
        
        if(n % 2 == 0) {// n이 짝수일 때
            cnt = n/2;
        } else {// n이 홀수일 때
            cnt = n/2 + 1;
        }
        
        int[] result = new int[cnt];
        int num = 1;
        
        for(int i = 0; i < result.length; i++) {
            result[i] = num;
            // 홀수 만들기
            num = num + 2;
        }
        
        return result;
    }
}

File: 프로그래머스/lv0/120813. 짝수는 싫어요/짝수는 싫어요.js
function solution(n) {
    var answer = [];
    
    // 1 ~ n까지 커지는 숫자 생성
    let cnt = 1;
    
    while(cnt <= n) {
        // cnt가 홀수일 때만
        if(cnt % 2 === 1){
            // 해당 값 배열에 넣어줌
            answer.push(cnt);
        }
        cnt = cnt + 1;
    }
    return answer;
}